packages/smart-contracts/src/contracts/ERC20TransferableReceivable.sol

```diff
@@ -31,6 +30,7 @@ contract ERC20TransferableReceivable is ERC721, ERC721Enumerable, ERC721URIStora
     address tokenAddress;
     uint256 amount;
     uint256 balance;
+    bytes32 requestID;
   }
 
   /**
```

Comment:
Change Request: 

Recommend adding a dev note for `receivableTokenIdMapping` (namely because this confused me for a while).

```
@dev Used by mint function to prevent minting multiple tokens for a given paymentReference and minter address
```

Reply:
I still think this would be a nice change. I wasn't able to add it as a suggestion because it's too far away from other edited lines.

Reply:
Sorry for not seeing this! Added